### PR TITLE
Update implementation experience

### PR DESCRIPTION
**Pull request by mengelbart**

### CODE REVIEW

**Comment by mengelbart** [comment]
@SpencerDawkins pointed me to [RFC 7942](https://datatracker.ietf.org/doc/html/rfc7942). Following the guidance from that RFC, I moved the section to the main document, included the template for each implementation, and added a note to remove the section before publication.

File: draft-ietf-avtcore-rtp-over-quic.md [hunk 1]
```diff
@@ -1193,6 +1213,147 @@ In addition, as noted in {{new-quic}}, one of the motivations for using QUIC as
...
+
+## mengelbart/roq
+
+Ogranization:
+: Technical University of Munich
+
+Implementation:
+: {{roq}}
+
+Descritption:
```
Comment (SpencerDawkins) [inline draft-ietf-avtcore-rtp-over-quic.md:1246]:
```suggestion
Description:
```

File: draft-ietf-avtcore-rtp-over-quic.md [hunk 2]
```diff
@@ -1193,6 +1213,147 @@ In addition, as noted in {{new-quic}}, one of the motivations for using QUIC as
...
+{{quic-go}} QUIC implementation.
+
+Level of Maturity:
+: prototype
+
+Coverage:
+: The library supports sending and receiving RTP and RTCP packets using QUIC
+streams and QUIC DATAGRAMs and multiplexing using flow identifiers. Applications
+using the library are responsible for appropriate signaling, setting up QUIC
+connections, and managing RTP sessions. Applications have to choose whether to
```
Comment (SpencerDawkins) [inline draft-ietf-avtcore-rtp-over-quic.md:1258]:
```suggestion
connections, and managing RTP sessions. Applications choose whether to
```

File: draft-ietf-avtcore-rtp-over-quic.md [hunk 3]
```diff
@@ -1193,6 +1213,147 @@ In addition, as noted in {{new-quic}}, one of the motivations for using QUIC as
...
+Implementation Experience:
+: The authors have no experience with the following:
+
+* Using the Multipath Extension for QUIC in combination with RoQ.
+* Influence of default priorities of QUIC implementations between streams and
+  DATAGRAMs on the performance of RTCP.
+* DATAGRAMs that are queued (and thus delayed) or dropped on expiration before
+  being transmitted due to congestion.
+* Translating between RoQ and RTP over UDP.
+* Performance impacts of multiplexing many sessions on a single QUIC connection.
```
Comment (SpencerDawkins) [inline draft-ietf-avtcore-rtp-over-quic.md:1279]:
```suggestion
: The implementer reports they have no experience with the following:

* Using RoQ with QUIC connections during path changes, whether due to QUIC connection migration or Multipath Extension for QUIC. As described in {{futures-new-ext}}, we expect future work on this topic.
* Influence of default priorities of QUIC implementations between streams and
  DATAGRAMs on the performance of RTCP. As described in {{futures-impl-deploy}}, we expect future work on this topic.
* DATAGRAMs that are queued (and thus delayed) or dropped on expiration before
  being transmitted due to congestion. As described in {{futures-impl-deploy}}, we expect future work on this topic
* Translating between RoQ and RTP over UDP. As described in {{topologies}}, RoQ can be used to connect to some RTP middleboxes using some topologies, and these middleboxes might be connecting RoQ endpoints and non-RoQ endpoints, so will need to translate between RoQ and RTP over UDP. 
* Performance impacts of multiplexing many RTP sessions on a single QUIC connection. As described in {{futures-impl-deploy}}, we expect future work on this topic
```
So, what I'm thinking is, this section could map pretty closely onto the rest of the document, and especially to {#futures}. If that makes sense, we might use {#futures} as a checklist, to name things that haven't been implemented or tested yet. 

Does that make sense?

File: draft-ietf-avtcore-rtp-over-quic.md [hunk 4]
```diff
@@ -1193,6 +1213,147 @@ In addition, as noted in {{new-quic}}, one of the motivations for using QUIC as
...
+: *roq* is a library implementing the basic encapsulation described in
+{{encapsulation}}. The library uses the Go programming language and supports the
+{{quic-go}} QUIC implementation.
+
+Level of Maturity:
+: prototype
+
+Coverage:
+: The library supports sending and receiving RTP and RTCP packets using QUIC
+streams and QUIC DATAGRAMs and multiplexing using flow identifiers. Applications
```
Comment (SpencerDawkins) [inline draft-ietf-avtcore-rtp-over-quic.md:1256]:
```suggestion
: The library supports sending and receiving RTP and RTCP packets using QUIC
streams and QUIC DATAGRAMs, and supports multiplexing using flow identifiers. Applications
```

**Review by SpencerDawkins — APPROVED** [review]
Nice work!

File: draft-ietf-avtcore-rtp-over-quic.md [hunk 5]
```diff
@@ -1193,6 +1217,139 @@ In addition, as noted in {{new-quic}}, one of the motivations for using QUIC as
...
+
+## bbc/gst-roq
+
+Ogranization:
+: BBC
+
+Implementation:
+: RTP-over-QUIC elements for GStreamer {{gst-roq}}
+
+Descritption:
```
Comment (SpencerDawkins) [inline draft-ietf-avtcore-rtp-over-quic.md:1291]:
```suggestion
Description:
```

File: draft-ietf-avtcore-rtp-over-quic.md [hunk 6]
```diff
@@ -1193,6 +1217,139 @@ In addition, as noted in {{new-quic}}, one of the motivations for using QUIC as
...
+
+## mengelbart/rtp-over-quic
+
+Ogranization:
+: Technical University of Munich
+
+Implementation:
+: RTP over QUIC {{RTP-over-QUIC}}
+
+Descritption:
```
Comment (SpencerDawkins) [inline draft-ietf-avtcore-rtp-over-quic.md:1323]:
```suggestion
Description:
```

**Review by SpencerDawkins — APPROVED** [review]
You handled my suggestion about pointing to {{futures}} perfectly. Thank you for that. 

I missed the other two "Descritption" nits - my bad. 

Now that I'm looking at an almost-empty Implementation Experience section, I wonder if you might add one or two points that would help someone else who wants to do their own implementation, that aren't explained in the body of the specification. One item might be to explain that RoQ relies on out-of-band solutions for connection setup, and since there's not a current RoQ SDP specification, you statically configured the connection information to allow testing. 

That might be useful if we wanted to ask the chairs to Request Publication while we're waiting on RoQ SDP.

**Pull request merged**